### PR TITLE
[New post] Generative AI apps with Amazon Bedrock - A getting started guide for Go developers

### PR DESCRIPTION
**Pull request by abhirockzz**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [x] The title of this pull request reflects the title of my content.
- [x] I have reviewed the [Content Review Checklist](https://github.com/build-on-aws/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [x] This pull request contains one piece of content (post or page).

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

**Comment by cherbk** [comment]
@mikegc-aws please indicate if you have bandwidth to review this piece. thanks

**Review by mikegc-aws — CHANGES_REQUESTED** [review]
Made notes and sent via Slack.

**Review by cherbk — APPROVED** [review]

File: concepts/amazon-bedrock-golang-getting-started/index.md [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+go run titan-text-embedding/main.go "cat"
+go run titan-text-embedding/main.go "dog"
+go run titan-text-embedding/main.go "trex"
+```
+
+This is probably the least exciting output you will see! The truth is, it's hard to figure out anything by looking at a slice of `float64`s 🤷🏽
+
+It is more relevant when combined with other components such as a Vector Database (for *storing* these embeddings) and use cases like semantic search (to *make use* of these embeddings). These topics will be covered in future blog posts - for now, just bear with the fact that "it works".
+
+## Closing thoughts
```
Comment (damonk10) [inline concepts/amazon-bedrock-golang-getting-started/index.md:408]:
```suggestion
## Closing Thoughts
```

File: concepts/amazon-bedrock-golang-getting-started/index.md [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+go run titan-text-embedding/main.go "trex"
+```
+
+This is probably the least exciting output you will see! The truth is, it's hard to figure out anything by looking at a slice of `float64`s 🤷🏽.
+
+It is more relevant when combined with other components such as a Vector Database (for *storing* these embeddings) and use cases like semantic search (to *make use* of these embeddings). These topics will be covered in future blog posts. For now, just bear with the fact that "it works".
+
+## Closing Thoughts
+
+I hope this proves useful for Go developers as a starting point on how to use Foundation models on Amazon Bedrock to power GenAI applications.
```
Comment (damonk10) [inline concepts/amazon-bedrock-golang-getting-started/index.md:410]: Please add two or three links to other pieces on next steps for the curious reader. What should they try next? What other information would be worth learning?

Comment (abhirockzz) [reply concepts/amazon-bedrock-golang-getting-started/index.md]: done

**Review by damonk10 — APPROVED** [review]

**Pull request merged**